advanced/Scripts/update.sh

```diff
@@ -35,25 +35,37 @@ source "/opt/pihole/COL_TABLE"
 
 GitCheckUpdateAvail() {
     local directory
+    local curBranch
     directory="${1}"
     curdir=$PWD
     cd "${directory}" || return
 
     # Fetch latest changes in this repo
     git fetch --quiet origin
```

Comment:
```suggestion
    git fetch --tags --quiet origin
```
This one is lacking `--tags`

`man git fetch` says:
> **-t, --tags**
> **Fetch all tags from the remote** (i.e., fetch remote tags refs/tags/* into local tags with the same name), **in addition to whatever else would otherwise be fetched**. Using this option alone does not subject tags to pruning, even if --prune is used (though tags may be pruned anyway if they are also the destination of an explicit refspec; see --prune).

(emphasis added by me). It sounds like a good idea to have it in addition.

Reply:
I did not include it because from https://git-scm.com/docs/git-fetch

> By default, any tag that points into the histories being fetched is also fetched; the effect is to fetch tags that point at branches that you are interested in. This default behavior can be changed by using the --tags or --no-tags options or by configuring remote.<name>.tagOpt. By using a refspec that fetches tags explicitly, you can fetch tags that do not point into branches you are interested in as well.

I concluded that any tags pointed to `master` are fetched. But I can include it just to be safe.